applications/crossbar/doc/websockets.md

```diff
@@ -0,0 +1,83 @@
+/*
+Section: Crossbar
+Title: Websockets
+Language: en-US
+*/
```

Comment:
Can you add a `Version` header so readers know what version this doc is current for?